sklearn/grid_search.py

```diff
@@ -326,17 +326,18 @@ def _check_param_grid(param_grid):
         param_grid = [param_grid]
 
     for p in param_grid:
-        for v in p.values():
+        for name, v in p.items():
             if isinstance(v, np.ndarray) and v.ndim > 1:
                 raise ValueError("Parameter array should be one-dimensional.")
 
             check = [isinstance(v, k) for k in (list, tuple, np.ndarray)]
             if True not in check:
-                raise ValueError("Parameter values should be a list.")
+                raise ValueError("Parameter ({0}) value should be a list."
```

Comment:
I have a hard time parsing this sentence. Currently when searching over parameter alpha, it says something like 

> Parameter (alpha) value should be a list.

That seems not very intuitive. Each parameter value for alpha should be a float.
Maybe " Parameter values for parameter {} need to be a sequence". ?